wikia/wikia.py

```diff
@@ -73,13 +73,17 @@ async def select_domain(ctx: commands.Context, pages: list, controls: dict, mess
         # Loop through subdomains to show them in menu
         for subdomains in data["items"]:
             embed = discord.Embed(color = await ctx.embed_color())
-            embed.title = subdomains["name"] + f" ({subdomains['url']})"
+            embed.title = subdomains["name"]
```

Comment:
Would like to have the link still there. Otherwise looks good, needs to be tested.